src/ansys/fluent/core/filereader/case_file.py

```diff
@@ -234,15 +234,15 @@ class Mesh:
     Methods
     -------
 
-    get_surface_ids
+    get_surface_ids()
         Get a list of surface ids.
-    get_surface_names
+    get_surface_names()
         Get a list of surface names.
-    get_surface_locs
+    get_surface_locs(surface_id)
```

Comment:
I noticed that `get_surface_locs()` is abbreviated differently from the other methods, which impacts the interface's consistency. Additionally, its name doesn’t fully capture the purpose described in the documentation. Perhaps `get_surface_location_bounds()` or `get_surface_location_indices()` could better convey its function, depending on what it's meant to return.

@prmukherj

Reply:
I agree @seanpearsonuk. I think the former makes better sense: ```get_surface_location_bounds()```

Reply:
For now, I'm listing these tasks in the documentation [super-issue](https://github.com/ansys/pyfluent/issues/3456).